lib/url.ts

```diff
@@ -35,7 +36,7 @@ export class URLQuery {
    * Get the value of the query parameter with the provided name. If no parameter exists with the
    * provided parameter name, then undefined will be returned.
    */
-  public get(parameterName: string): string | undefined {
+  public get(parameterName: string): string | Array<any> | undefined {
```

Comment:
`any[]` instead of `Array<any>`.

Reply:
This method usually just indexes into _rawQuery so it seems reasonable to make it a `string | string[] | undefined`.